tests/src/JIT/Performance/CodeQuality/Span/SpanBench.cs

```diff
@@ -717,12 +953,23 @@ public static void TestSpanClearByte(int length)
         [InlineData(1000)]
         public static void TestSpanClearString(int length)
         {
-            var array = new string[length];
-            var span = new Span<string>(array);
-            foreach (var iteration in Benchmark.Iterations)
-                using (iteration.StartMeasurement())
-                    for (int i = 0; i < Benchmark.InnerIterationCount; i++)
-                        span.Clear();
+            InvokeTestSpanClear<string>(length);
+        }
+
+        static void InvokeTestSpanClear<T>(int length)
+        {
+            var array = new T[length];
+
+            Invoke((int innerIterationCount) => TestSpanClear<T>(array, innerIterationCount),
+                "TestSpanClear<{0}>({1})", typeof(T).Name, length);
+        }
+
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        static void TestSpanClear<T>(T[] array, int iterationCount)
+        {
+            var span = new Span<T>(array);
+            for (int i = 0; i < iterationCount; i++)
```

Comment:
Measures construction + clear. Is the hopes/goal that since we clear "iterationCount" times that it would dwarf the constructor time? If not, when comparing to Array.Clear, span has a disadvantage.